### PR TITLE
unicorn-python: remove package

### PR DESCRIPTION
**Pull request by paper42**
@Piraty 

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/removed-packages/template [hunk 1]
```diff
@@ -340,6 +341,7 @@ replaces="
  traverso<=0.49.6_2
  ttyload-git<=20141117_4
  tuxc<=1.1_3
+ unicorn<=1.0.2_2
```
Comment (abenson) [inline srcpkgs/removed-packages/template:344]: Why are you replacing `unicorn`?

File: srcpkgs/unicorn/template [hunk 2]
```diff
@@ -2,7 +2,7 @@
 pkgname=unicorn
 version=1.0.2
 revision=2
```
Comment (abenson) [inline srcpkgs/unicorn/template:4]: This would need a revbump.

File: srcpkgs/removed-packages/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/removed-packages/template]: oh, thank you for catching this!

File: srcpkgs/unicorn/template (hunk 2 above)
Comment (paper42) [reply srcpkgs/unicorn/template:4]: As we discussed on IRC, it shouldn't be necessary because all remaining subpackages will still install the same files.

**Comment by Piraty** [comment]
fine

**Pull request merged**